editor/components/PointMap.tsx

```diff
@@ -128,6 +129,9 @@ export const PointMap = ({
       onDrag={isEditing ? onDrag : undefined}
       maxZoom={21}
     >
+      {basemapType === "aerial" && (
+        <MapAerialSourceAndLayer></MapAerialSourceAndLayer>
```

Comment:
Just a stylistic preference, but could this be `<MapAerialSourceAndLayer />` or will there eventually be child components?